p4-16/spec/P4-16-spec.mdk

```diff
@@ -3732,39 +3732,39 @@ finding this information in a section dedicated to type `varbit`.
 
 P4 provides a limited set of casts between types. A cast is written
 `(t) e`, where `t` is a type and `e` is an expression. Casts are only
-permitted on base types and derived types introduced by `typedef`, `type`, and `enum`. While this design is arguably more onerous
-for programmers, it has several benefits:
+permitted on base types and derived types introduced by `typedef`, `type`, and `enum`.
+While this design is arguably more onerous for programmers, it has several benefits:
 
 - It makes user intent unambiguous.
 - It makes the costs associated with converting numeric values
-  explicit. Implementing certain casts involve sign-extensions, and
-  thus can require significant computational resources on some
-  targets.
+  explicit. Implementing certain casts involve sign extensions,
```

Comment:
Why not "involves"?

Reply:
A pre-Abe bug I did _not_ catch right next to a bug I _did_ catch.  Thanks.  Adding a fix in my private repo. (should be included in my next PR).